doc/howtos.rst

```diff
@@ -45,7 +45,22 @@ And the Python update triggers in ``debian/«pkgname».triggers`` need to be ada
     interest-noawait /usr/bin/python3.5
     …
 
-That's all.
+You may also need to add the ``--python`` flag in ``debian/rules``.
+
+.. code-block:: make
+
+    %:
+        dh $@ --with python-virtualenv --python python3
```

Comment:
This is not working for me.
I have to specify `--python python3` here instead:
```
%:
	dh $@ --with python-virtualenv

override_dh_virtualenv:
	dh_virtualenv --python python3
```

Reply:
Seems you're correct. Looking through the history of one of my projects, it seems to be what I did too, actually. I had already switched to the `--buildsystem` approach when I wrote this piece of documentation, so I must've gotten it mixed up. I'll change it.